### PR TITLE
kernel: Note prereqs in document

### PR DESCRIPTION
**Pull request by grahamwhaley**
Note that to use the build script you need to have some prereqs,
including a new enough golang.

Reported-by: Rory Savage <rsavage@dispersivegroup.com>
Signed-off-by: Graham Whaley <graham.whaley@intel.com>

### CODE REVIEW

**Comment by grahamwhaley** [comment]
Replaces the stuck #310

**Comment by grahamwhaley** [comment]
doh, forgot the Fixes - will update!

**Comment by jodh-intel** [comment]
Extra points for adding a ToC on another commit ;)

**Comment by grahamwhaley** [comment]
ToC is done
/me hands himself 🍰

File: kernel/README.md [hunk 1]
```diff
@@ -1,9 +1,25 @@
+* [Build Kata Containers Kernel](#build-kata-containers-kernel)
+    * [Requirements](#requirements)
+    * [Setup kernel source code](#setup-kernel-source-code)
+* [Build the kernel](#build-the-kernel)
+    * [Install the Kernel in the default path for Kata](#install-the-kernel-in-the-default-path-for-kata)
+    * [Submit Kernel Changes](#submit-kernel-changes)
+    * [How is it tested.](#how-is-it-tested)
```
Comment (jodh-intel) [inline kernel/README.md]: Nit: period in heading.

Comment (grahamwhaley) [reply kernel/README.md]: heh, guess you'd like me to fix that as well? :-) The auto toc generator put it in... ;-)

Comment (jodh-intel) [reply kernel/README.md]: I know it's technically outside the scope of this PR, but maybe one tiny extra commit to remove the period in the heading before your commit...?  :)

Comment (grahamwhaley) [reply kernel/README.md]: See if the '.' is done now...

**Comment by jodh-intel** [comment]
@grahamwhaley - thanks.

**Comment by jcvenegas** [comment]
@grahamwhaley seems that you need to bump the kernel version file we can improve the changes detection to ignore documentation files.

**Pull request merged**